website/static/js/fangorn.js

```diff
@@ -1691,8 +1691,7 @@ tbOptions = {
         var item = tb.find(row.id);
         _fangornMultiselect.call(tb,null,item);
     },
-    hScroll : 400,
-    naturalScrollLimit : 1000
+    hScroll : 400
```

Comment:
@caneruguz Does this also need to be changed for the presenation service treebeard?